apps/mobile/src/components/ChatDetailScreen/components/VexlbotMessageItem/components/TradeChecklistDateAndTimeView.tsx

```diff
@@ -24,7 +33,30 @@ export default function TradeChecklistDateAndTimeView(): JSX.Element | null {
         text={`${t('vexlbot.yourMeetingIsOn')}\n${dateAndTime.toStringWithTime(
           pick.pick.dateTime
         )}`}
-      ></VexlbotBubble>
+      >
+        <Button
+          onPress={() => {
+            void addEventToCalendar({
```

Comment:
What if we already know the event location here (it's sent after they agreed on location). Why not add it, if it is set